### DISCUSSION TITLE
该有的功能都有，值得一个星星！

### DISCUSSION THREAD

**Discussion by Flyingpen** [Thysrael/Ficus#241] [General]
知乎来的，下载使用了一下，界面简约，功能上该有的都有，无须像obsidian那样配置一堆插件，很赞。但也因为这个原因，功能上并没有太丰富。我个人感觉的唯一缺点，对markdown新手没有太友好，编辑工具栏要通过反复点设置才能实现。
希望能够集成一下，给个固定或者跟随的工具栏，毕竟不是每个作者都喜欢用代码，有时候鼠标点一点也很方便呐。

**Comment by Thysrael** [comment]
感谢反馈，确实是这个问题，我们会考虑加一个跟随工具栏的
